src/applications/vaos/tests/new-appointment/components/DateTimeRequestPage.unit.spec.jsx

```diff
@@ -1,213 +1,347 @@
 import React from 'react';
 import { expect } from 'chai';
 import sinon from 'sinon';
-import { mount } from 'enzyme';
 import moment from 'moment';
+import { renderWithStoreAndRouter } from '../../mocks/setup';
+import userEvent from '@testing-library/user-event';
 
 import {
   DateTimeRequestPage,
```

Comment:
This import is getting the page component without the Redux logic attached to it. When rewriting tests with RTL we've mostly been trying to test the component when connected to Redux, so that it's a more realistic test. To do that, you'd get the default import for that component, instead of these two named imports.

This would mean a few changes in our tests below, mainly that you won't pass props directly to DateTimeRequestPage, you'll set data in the Redux store.

You can see an example of that here: https://github.com/department-of-veterans-affairs/vets-website/blob/master/src/applications/vaos/tests/new-appointment/components/ConfirmationPage/index.unit.spec.jsx#L26

You should need to mock less of the Redux store than that test.